src/test/query_test.cc

```diff
@@ -298,7 +298,8 @@ TEST_F(QueryTest, NoQueryFn)
   ASSERT_TRUE(err != nullptr) << "Expect error";
   EXPECT_EQ(TRITONSERVER_ErrorCode(err), TRITONSERVER_ERROR_UNAVAILABLE);
   EXPECT_STREQ(
-      TRITONSERVER_ErrorMessage(err), "Output properties are not available");
+      TRITONSERVER_ErrorMessage(err),
+      "[request id: <id_unknown>] Output properties are not available");
```

Comment:
To your comment about gtest not supporting substring easily, I think you can do something like this:
```
using ::testing::HasSubstr;  // I think you'll need this up top or somewhere similar

EXPECT_THAT(
  TRITONSERVER_ErrorMessage(err), 
  HasSubstr("Output properties are not available")
);
```